stories/index.stories.js

```diff
@@ -262,35 +264,28 @@ export const S1GRDFindTiles = () => {
 
 export const findFlyovers = () => {
   const layer = new S2L2ALayer(instanceId, s2l2aLayerId);
-  const bbox = new BBox(CRS_EPSG4326, 11.9, 42.05, 12.95, 43.09);
 
   const wrapperEl = document.createElement('div');
-  wrapperEl.innerHTML = "<h2>findFlyovers</h2>";
+  wrapperEl.innerHTML = '<h2>findFlyovers</h2>';
```

Comment:
perhaps add S2/Sentinel 2 to heading  and function name ( both S1 and S2 are used in this story).

Reply:
Thanks, added.